CMakeLists.txt

```diff
@@ -337,6 +337,24 @@ IF(NOT CYCLUS_DOC_ONLY)
                         OUTPUT_VARIABLE core_version
                         OUTPUT_STRIP_TRAILING_WHITESPACE)
     ENDIF(NOT "${CORE_VERSION}" STREQUAL "")
+
```

Comment:
These changes to CMakeLists.txt do not belong in this pull request. The pull request should be from a clean develop branch. As is, you have changes that apply to both issue #1369 and issue #1374 in this pull request. 

These three commits do not belong in this PR:
- 3c59474
- 303400c
 - 6468d9b

This commit is the only commit that should be included in this PR: c98440b